### ISSUE TITLE
Optifine + AccessTweaks

### ISSUE THREAD

**Issue by Mario0051** [JayAvery/accesstweaks#2]
The AccessTweaks video settings override the Optifine settings, the Optifine settings can only be changed with the optionsof.txt file and there is no other way to modify the settings.

**Comment by averixus** [comment]
AccessTweaks replaces some vanilla classes by necessity, so it may not be possible to be compatible with Optifine. Which settings interfere with each other?

**Comment by Mario0051** [comment]
The optfine video settings are completely overridden and replaced with your video settings, but it is still compatible. The video settings of optifine are able to be modified with the optionsof.txt file and the settings still work, just that you cannot access them in-game.

**Comment by averixus** [comment]
Sorry, I'm not sure I understand. Do you mean the GUI for adjusting Optifine's settings is inaccessible in game? Or the actual changes Optifine makes are ignored?

**Comment by Mario0051** [comment]
Yes, the GUI for adjusting Optifine's settings is inaccessible in game.

**Comment by averixus** [comment]
I'm afraid I have no idea why that is. AccessTweaks doesn't intentionally interfere with any GUI stuff, and I can't think of a reason why it would make Optifine settings inaccessible. As Optifine is closed source, there's not really anything more I can do to figure out the problem.
